### PR TITLE
Add generator for Eslint

### PR DESCRIPTION
**Pull request by avdgaag**
This adds a generator for quickly adding Eslint to a project. It will install the package, copy default configuration and add NPM and Rake tasks.

@florish you've used these generators most recently. Can you give them a try in one of your projects?
@stefanroex You've built the most projects with eslint in them, I think. These default settings are taken from just one project; do you have any suggestions for sane defaults?

### CODE REVIEW

**Comment by stefanroex** [comment]
@avdgaag I've super happy with the [Airbnb](https://github.com/airbnb/javascript/tree/master/packages/eslint-config-airbnb) defaults. I've only used this in combination with ES6 + React, so I'm not sure how they behave in other types of projects.

**Comment by avdgaag** [comment]
> I've super happy with the Airbnb defaults. I've only used this in combination with ES6 + React, so I'm not sure how they behave in other types of projects.

Then I'm tempted to _not_ make them the defaults. Once you have eslint set up, it is kind of trivial to import other settings. If it turns out we do want to use those ALL THE TIME, then we can always add them as defaults later on.

File: brightin-template [hunk 1]
```diff
@@ -185,19 +186,21 @@ module BrightinTemplate
...
     end
 
+    def install_npm_package
+      if npm? && options[:install]
+        system 'npm install --save-dev coffeelint'
+      else
+        say "You should install coffeelint globally:\n\n  npm install -g coffeelint"
+      end
+    end
+
```
Comment (florish) [inline brightin-template]: @avdgaag These `coffeelint` adjustments seem to be unrelated to `eslint`, and a first glance, the changes here seem to be incomplete? Maybe I'm missing something, just wanted to let you know.

**Comment by florish** [comment]
@avdgaag I don't have any projects available with ES code (as in, not CoffeeScript or "traditional" JS) which do not already have `eslint` installed at this moment. This means I'm not able to test try this out for you right now. Will keep it in mind for new projects!

**Comment by florish** [comment]
As for the code: apart from my comment above, everything looks fine 👍

File: brightin-template (hunk 1 above)
Comment (avdgaag) [inline brightin-template]: Meh, who uses coffee script anyway? ¯\_(ツ)_/¯
Anyhoo, I probably should've split this into two PRs. But it all seems to work for me. What do you think is missing?

Comment (florish) [inline brightin-template]: Couldn't find a method definition for `npm?` and at which `install_npm_package` is called during install, but I think I overlooked `include Helpers`. Never mind!
